src/pages/case-study/central-working-changing-how-you-work.js

```diff
@@ -0,0 +1,428 @@
+import React, { Fragment } from 'react'
+import { graphql } from 'gatsby'
+import styled from 'styled-components'
+import breakpoint from 'styled-components-breakpoint'
+import ReactMarkdown from 'react-markdown'
+import { generate } from 'shortid'
+
+import { Grid, Row, Col } from '../../components/grid'
+import GreyBackground from '../../components/Common/GreyBackground'
+import BrownBackground from '../../components/Common/BrownBackground'
+import TanBackground from '../../components/Common/TanBackground'
+
+import Layout from '../../components/layout'
+import Head from '../../components/Common/Head'
+import Hr from '../../components/Common/Hr'
+import CaseStudyHero from '../../components/Common/CaseStudyCards/CaseStudyHero'
+import { SectionTitle, BodyPrimary } from '../../components/Typography'
+import { TextColumnsBlock } from '../../components/Common/CaseStudyCards/CaseStudyBlocks'
+
+import CaseStudyPreview from '../../components/Common/CaseStudyCards/CaseStudyPreview'
+import Image from '../../components/Common/Image'
+
+const shouldRender = data => data && data.length
+
+const getImage = (blockImages, index) => blockImages && blockImages[index]
+
+const normaliseAll = (arr = []) =>
+  arr.map(({ genericBlockText, genericBlockImages, ...props }) => ({
+    image: getImage(genericBlockImages, 0),
+    text: genericBlockText && genericBlockText.genericBlockText,
+    ...props
+  }))
+
+const normalise = (arr = [], index = 0) => normaliseAll(arr, index)[index]
+
+const GreyBodyPrimary = styled(BodyPrimary)`
+  color: ${({ theme }) => theme.colors.secondaryText};
+`
+
+const Block3Col = styled(Col)`
```

Comment:
Block3Col, 4 and 6 are exactly the same. I would rename to something like BlockColPaddingTop.

Reply:
Only 4 and 6 are the same no? 

I would have created a styled component here but not sure what to call it. `BlockColPaddingTop` is super vague in this context.